### PR TITLE
Feature/713

### LINKED ISSUE #713: Disable edit mode buttons in week plan overview when user has not selected any week plans
**Is your feature request related to a problem? Please describe.**
When no week plan has been selected by the user, the buttons in the bottom bar are active anyway.

**Describe the solution you'd like**
The buttons should be deactivated when the user is in edit mode and hasn't selected any week plans yet.

**Additional context**
These buttons should be disabled:
Screenshot_1607524237

### PR DESCRIPTION
**Pull request by ChristofferAAU**
Made it so that the buttons are unresponsive when a weekplan is not selected.

### CODE REVIEW

**Comment by JakobFaarGreg** [comment]
@JakobFaarGreg 's tjekliste

- [x] The environments.json file has the following default values:
{
  "SERVER_HOST": "https://srv.giraf.cs.aau.dk/DEV/API",
  "DEBUG": true,
  "USERNAME": "Graatand",
  "PASSWORD": "password"
}


Code Design 

- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [x] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [x] This code could not, to the best of my knowledge, have reused existing code
- [x] The code does not introduce functionality that is unnecessary for solving the problem

Code Readability 

- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [x] The code is simple and readable. i.e. it contains no "hack", or obscure solution

Code Maintainability

- [x] Has all the added code been properly commented

Only check one of the items below:
- [x] The functionality does not need to be covered by tests
- [ ] The functionality needs to be covered by tests

Only check these if the functionality needs to be covered by tests:

- [ ] All functionality is covered by tests
- [ ] The tests has sensible names
- [ ] By reading the tests you know what they cover
- [ ] The tests cover sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing

Functionality

For this part, open the app, and test:
- [x] The features that the code claims to implement, are actually implemented
- [x] The code does not look like it has bugs
- [x] All the new screens are reachable through in-app navigation

**Comment by JakobFaarGreg** [comment]
#713

**Review by JakobFaarGreg — DISMISSED** [review]
Everything looks good to me.

File: lib/screens/weekplan_selector_screen.dart [hunk 1]
```diff
@@ -420,6 +425,9 @@ class WeekplanSelectorScreen extends StatefulWidget {
                 description: 'Der skal markeres præcis én uge for at kopiere');
           });
     }
+    if(widget._weekBloc.getNumberOfMarkedWeekModels() < 1){
```
Comment (HowToYEET) [inline lib/screens/weekplan_selector_screen.dart:425]: begge sider af condition mangler mellemrum

**Review by HowToYEET — CHANGES_REQUESTED** [review]

File: lib/screens/weekplan_selector_screen.dart (hunk 1 above)
Comment (ChristofferAAU) [reply lib/screens/weekplan_selector_screen.dart:425]: Done

**Review by HowToYEET — DISMISSED** [review]

File: lib/screens/login_screen.dart [hunk 2]
```diff
@@ -123,6 +123,7 @@ class LoginScreenState extends State<LoginScreen> {
     final bool keyboard = MediaQuery.of(context).viewInsets.bottom > 0;
 
     return Scaffold(
+      //resizeToAvoidBottomPadding: false, --Fix this
```
Comment (JakobFaarGreg) [inline lib/screens/login_screen.dart]:
```suggestion
    
```

File: lib/screens/weekplan_selector_screen.dart [hunk 3]
```diff
@@ -69,7 +68,7 @@ class WeekplanSelectorScreen extends StatefulWidget {
           builder: (BuildContext context, AsyncSnapshot<bool> snapshot) {
             if (snapshot.data) {
               return _buildBottomAppBar(context);
-            } else {
+            }else {
```
Comment (JakobFaarGreg) [inline lib/screens/weekplan_selector_screen.dart]:
```suggestion
            } else {
```

File: lib/screens/weekplan_selector_screen.dart [hunk 4]
```diff
@@ -352,7 +351,8 @@ class WeekplanSelectorScreen extends StatefulWidget {
                       text: 'Redigér',
                       icon:
                           const ImageIcon(AssetImage('assets/icons/edit.png')),
-                      onPressed: () async => _pushEditWeekPlan(context)),
+                      onPressed: () async => _pushEditWeekPlan(context),
```
Comment (JakobFaarGreg) [inline lib/screens/weekplan_selector_screen.dart]:
```suggestion
                      onPressed: () async => _pushEditWeekPlan(context)
```

**Review by JakobFaarGreg — CHANGES_REQUESTED** [review]
Jeg har et par små ændringer der gerne skal fixes.

**Comment by ChristofferAAU** [comment]
> Jeg har et par små ændringer der gerne skal fixes.

Burde være fikset nu

**Review by JakobFaarGreg — APPROVED** [review]

**Review by HowToYEET — APPROVED** [review]

**Pull request merged**